proxy/wpr.py

```diff
@@ -0,0 +1,74 @@
+import os
+import signal
+import subprocess
+import sys
+
+from mozprocess import ProcessHandler
+from proxy.mitmproxy import MITMProxy404
+
+dirname = os.path.dirname(__file__)
+
+
+class Wpr(object):
+    def __init__(self, path, record=True):
+        self.path = path
+        self.record = record
+        self.binary_path = os.path.join(dirname, "utils", "wpr")
+        self.scripts = os.path.join(dirname, "scripts")
+
+    def __enter__(self):
+        self.port_fw =  MITMProxy404(path=self.path, port_fw=True)
+        self.port_fw.start()
+
+        print "Staring WPR"
+        print " ".join(self.command())
+
+        self.process = ProcessHandler(self.command())
+                                      # logfile=os.path.join(dirname, "wpr.log"))
```

Comment:
Can we use subprocess, and restore the logging?

Reply:
Done